### PR TITLE
Update direct-routing-plan.md

### PR DESCRIPTION
**Pull request by NMuravlyannikov**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/direct-routing-plan.md [hunk 1]
```diff
@@ -299,6 +299,26 @@ The port range of the Media Processors is shown in the following table:
   > [!NOTE]
   > Microsoft recommends at least two ports per concurrent call on the SBC.
 
+
+## Media traffic: Media Processors geography
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
## Media traffic: Media processors geography
```

File: Teams/direct-routing-plan.md [hunk 2]
```diff
@@ -299,6 +299,26 @@ The port range of the Media Processors is shown in the following table:
...
+
+The media traffic flows via components, called Media Processors. 
+Media Processors placed in the same datacenters as SIP Proxies. Also, there are additional media processors to optimize the media flow. For example, now we do not have a SIP Proxy component in Australia (SIP flows via Singapore or Hong Kong) but we do have the Media Processor locally in Australia. Need of the Media Processors locally dictated by the latency which we  experience by sending traffic on long distance, for example from Australia to Singapore or Hong Kong. While latency in the example of traffic flowing from Australia to Hong Kong or Singapore is acceptable to preserve good call quality for SIP traffic, for Real Time Media traffic it is not.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
Media processors are placed in the same datacenters as SIP proxies. Also, there are additional media processors to optimize media flow. For example, now we do not have a SIP proxy component in Australia (SIP flows via Singapore or Hong Kong) but we do have the media processor locally in Australia. The need for the media processors locally is dictated by the latency which we experience by sending traffic long-distance, for example from Australia to Singapore or Hong Kong. While latency in the example of traffic flowing from Australia to Hong Kong or Singapore is acceptable to preserve good call quality for SIP traffic, for real-time media traffic it is not.
```

File: Teams/direct-routing-plan.md [hunk 3]
```diff
@@ -299,6 +299,26 @@ The port range of the Media Processors is shown in the following table:
   > [!NOTE]
   > Microsoft recommends at least two ports per concurrent call on the SBC.
 
+
+## Media traffic: Media Processors geography
+
+The media traffic flows via components, called Media Processors.
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
The media traffic flows via components called media processors. 
```

File: Teams/direct-routing-plan.md [hunk 4]
```diff
@@ -299,6 +299,26 @@ The port range of the Media Processors is shown in the following table:
   > [!NOTE]
   > Microsoft recommends at least two ports per concurrent call on the SBC.
 
+
+## Media traffic: Media Processors geography
+
+The media traffic flows via components, called Media Processors. 
+Media Processors placed in the same datacenters as SIP Proxies. Also, there are additional media processors to optimize the media flow. For example, now we do not have a SIP Proxy component in Australia (SIP flows via Singapore or Hong Kong) but we do have the Media Processor locally in Australia. Need of the Media Processors locally dictated by the latency which we  experience by sending traffic on long distance, for example from Australia to Singapore or Hong Kong. While latency in the example of traffic flowing from Australia to Hong Kong or Singapore is acceptable to preserve good call quality for SIP traffic, for Real Time Media traffic it is not.
+
+Location of the Media Processors:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
Location of the media processors:
```

File: Teams/direct-routing-plan.md [hunk 5]
```diff
@@ -299,6 +299,26 @@ The port range of the Media Processors is shown in the following table:
...
+
+Locations where both SIP proxy and Media Processor components deployed:
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
Locations where both SIP proxy and media processor components deployed:
```

File: Teams/direct-routing-plan.md [hunk 6]
```diff
@@ -299,6 +299,26 @@ The port range of the Media Processors is shown in the following table:
...
+	- US (two in US west and US East datacenters)
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
	- US (two in US West and US East datacenters)
```

File: Teams/direct-routing-plan.md [hunk 7]
```diff
@@ -299,6 +299,26 @@ The port range of the Media Processors is shown in the following table:
...
+	- Europe (Amsterdam and Dublin datacenters)
+	- Asia (Singapore and Hong Kong datacenters)
+
+Locations where only Media Processors deployed (SIP flows via nearest datacenter listed above)
```
Comment (JohanFreelancer9) [inline Teams/direct-routing-plan.md]:
```suggestion
Locations where only media processors are deployed (SIP flows via the closest datacenter listed above):
```

**Pull request merged**